Woof/WoofTests/ViewModel/SitterProfileViewModelTests/SitterProfileViewModelTests.swift

```diff
@@ -1,14 +1,37 @@
+import NetworkService
 import XCTest
 
 final class SitterProfileViewModelTests: XCTestCase {
+    override func setUp() {
+        super.setUp()
+        URLProtocol.registerClass(MockURLProtocol.self)
+    }
+
+    override func tearDown() {
+        URLProtocol.unregisterClass(MockURLProtocol.self)
+        super.tearDown()
+    }
+
     func testSitterProfileViewModelCanBeInitialized() {
         _ = SitterProfileViewModel()
     }
 
-    func testSitterProfileViewModelInitializedWithLastSavedData() {
+    func testSitterProfileViewModelInitializedWithLastSavedData() async {
         // Given
         let savingSitter = SitterProfileViewModel()
```

Comment:
The naming is confusing
```suggestion
        let savingSitterViewModel = SitterProfileViewModel()
```

Reply:
Fixed with a618ed9